Readme.md

```
@@ -487,7 +487,7 @@ gg "fix login bug" --osc
 
 gg "fix login bug" --osc --genie
 
-#### Basic Usage:
+#### Basic Usage
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)